sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -534,92 +632,95 @@ def send_message(self, message, **kwargs):
+    :paramtype prefetch: int
+    :keyword max_frame_size: Maximum AMQP frame size. Default is 63488 bytes.
+    :paramtype max_frame_size: int
+    :keyword channel_max: Maximum number of Session channels in the Connection.
+    :paramtype channel_max: int
+    :keyword idle_timeout: Timeout in seconds after which the Connection will close
      if there is no further activity.
-    :type idle_timeout: int
-    :param properties: Connection properties.
-    :type properties: dict
-    :param remote_idle_timeout_empty_frame_send_ratio: Ratio of empty frames to
+    :paramtype idle_timeout: int
+    :keyword properties: Connection properties.
+    :paramtype properties: dict
+    :keyword remote_idle_timeout_empty_frame_send_ratio: Ratio of empty frames to
      idle time for Connections with no activity. Value must be between
      0.0 and 1.0 inclusive. Default is 0.5.
-    :type remote_idle_timeout_empty_frame_send_ratio: float
-    :param incoming_window: The size of the allowed window for incoming messages.
-    :type incoming_window: int
-    :param outgoing_window: The size of the allowed window for outgoing messages.
-    :type outgoing_window: int
-    :param handle_max: The maximum number of concurrent link handles.
-    :type handle_max: int
-    :param on_attach: A callback function to be run on receipt of an ATTACH frame.
+    :paramtype remote_idle_timeout_empty_frame_send_ratio: float
+    :keyword incoming_window: The size of the allowed window for incoming messages.
+    :paramtype incoming_window: int
+    :keyword outgoing_window: The size of the allowed window for outgoing messages.
+    :paramtype outgoing_window: int
+    :keyword handle_max: The maximum number of concurrent link handles.
+    :paramtype handle_max: int
+    :keyword on_attach: A callback function to be run on receipt of an ATTACH frame.
      The function must take 4 arguments: source, target, properties and error.
-    :type on_attach: func[~uamqp.address.Source, ~uamqp.address.Target, dict, ~uamqp.errors.AMQPConnectionError]
-    :param encoding: The encoding to use for parameters supplied as strings.
+    :paramtype on_attach: func[~pyamqp.endpoint.Source, ~pyamqp.endpoint.Target, dict, ~pyamqp.errors.AMQPConnectionError]
+    :keyword encoding: The encoding to use for parameters supplied as strings.
      Default is 'UTF-8'
-    :type encoding: str
+    :paramtype encoding: str
     """
 
     def __init__(self, hostname, source, auth=None, **kwargs):
         self.source = source
-        self._streaming_receive = kwargs.pop("streaming_receive", False)  # TODO: whether public?
+        self.streaming_receive = kwargs.pop("streaming_receive", False) 
         self._received_messages = queue.Queue()
-        self._message_received_callback = kwargs.pop("message_received_callback", None)  # TODO: whether public?
+        self.message_received_callback = kwargs.pop("message_received_callback", None) 
```

Comment:
I don't think these two should really be public - what was the reasoning?

Reply:
I will change them to be private, I thought we had talked about pyamqp kwargs being public based off of the meeting notes.

Reply:
I remember talking about making them public during our initial meeting to match uamqp, but could be misremembering and that reflects in the meeting notes. So this one is on me :)